index.html

```diff
@@ -1050,8 +1053,31 @@ <h3>Processing Model</h3>
 <li>Record the initiator of the resource in
 <a>initiatorType</a>.</li>
 <li>Record the <a data-cite="HTML#resolve-a-url">resolved URL</a>
-of the requested resource in <a>name</a>. If there is an
-<a data-cite=
+  of the requested resource in <a>name</a>.</li>
+<li><em>Fetch start:</em> <dfn data-lt="step-fetch-start">Immediately before a
+user agent starts the <a data-cite="FETCH#concept-fetch" data-lt=
+'fetch'>fetching process</a></dfn>, record the <a>current time</a>
+as <a>fetchStart</a>. Let <a>domainLookupStart</a>,
+<a>domainLookupEnd</a>, <a>connectStart</a> and <a>connectEnd</a>
+be the same value as <a>fetchStart</a>.</li>
+<li><dfn data-lt="step-collection-start">If the user agent is to
+reuse the data from another existing or completed <a data-cite=
+"FETCH#concept-fetch">fetch</a> initiated from the <a>current
+document</a></dfn>, abort the remaining steps.</li>
+<li>If the resource fails the <a>timing allow check</a> algorithm, the user
+agent MUST run the following substeps:
+  <ol>
+    <li>If the resource's <a>Request</a>'s
```

Comment:
This logic is still leaving workerStart unset when the TAO check fails and destination is not "document". Perhaps we should move the service worker step so that it occurs before this step? This way, when TAO fails, we just overwrite the workerStart value to 0 when needed (and keep the computed value otherwise). Alternatively, create a var representing the TAO check. Set workerStart using this variable and the destination logic. Then, have the step that sets 0's and skips when this check fails.

Reply:
Forgot to bail if the destination is not document...